src/items/class-features/greater_resistance_(ex_drone).json

```diff
@@ -0,0 +1,102 @@
+{
+  "_id": "6pdTGIiXK4kUnBOG",
+  "name": "Greater Resistance (Ex; Drone)",
+  "type": "mod",
+  "img": "systems/sfrpg/icons/classes/greater_resistance.webp",
+  "system": {
+    "type": "",
+    "ability": null,
+    "abilityMods": {
+      "parts": []
+    },
+    "actionTarget": "",
+    "actionType": "",
+    "activation": {
+      "type": "",
+      "condition": "",
+      "cost": 0
+    },
+    "additionalMovement": "",
+    "additionalSenses": "",
+    "area": {
+      "effect": "",
+      "shape": "",
+      "units": "",
+      "value": 0
+    },
+    "arms": {
+      "armType": "general",
+      "number": null
+    },
+    "attackBonus": 0,
+    "bonusSkill": "",
+    "chatFlavor": "",
+    "critical": {
+      "effect": "",
+      "parts": []
+    },
+    "damage": {
+      "parts": []
+    },
+    "damageNotes": "",
+    "description": {
+      "chat": "",
+      "gmnotes": "",
+      "short": "",
+      "unidentified": "",
+      "value": "<p>&gt;@UUID[Compendium.sfrpg.class-features.cQtmz297P5g6g5xv]{Advanced Mods}</p>\n<p>Choose one type of energy for which your drone already has the resistance mod. Increase your drone’s resistance to that energy type by 5. You can select this mod multiple times. Its effects do not stack. Each time you select this mod, it applies to a different energy type for which your drone has the resistance mod.</p>\n<p>@UUID[Compendium.sfrpg.class-features.wJcZkzm1cs8aZhPc]{Resistance (Ex; Drone)}</p>\n<p><em>Need to select the appropriate resistance in the Modifiers tab.</em></p>"
```

Comment:
Notes should always be an instruction, rather than a "need".
"Select the appropriate resistance in the Modifiers tab."